va/va_enc_hevc.h

```diff
@@ -45,6 +45,200 @@ extern "C" {
+         * including block sizes which may be further constrained as
+         * noted in the VAConfigAttribEncHEVCBlockSizes attribute.
+         */
+        uint32_t pcm                        : 2;
+        /** Temporal motion vector Prediction.
+         *
+         * Allows setting slice_temporal_mvp_enabled_flag in slice
+         * headers.
+         */
+        uint32_t temporal_mvp               : 2;
+        /** Strong intra smoothing.
+         *
+         * Allows setting strong_intra_smoothing_enabled_flag in the SPS.
+         */
+        uint32_t strong_intra_smoothing     : 2;
+        /** Dependent slices.
+         *
+         * Allows setting dependent_slice_segment_flag in slice headers.
+         */
+        uint32_t dependent_slices           : 2;
+        /** Sign data hiding.
+         *
+         * Allows setting sign_data_hiding_enable_flag in the PPS.
+         */
+        uint32_t sign_data_hiding           : 2;
+        /** Constrained intra prediction.
+         *
+         * Allows setting constrained_intra_pred_flag in the PPS.
+         */
+        uint32_t constrained_intra_pred     : 2;
+        /** Transform skipping.
+         *
+         * Allows setting transform_skip_enabled_flag in the PPS.
+         */
+        uint32_t transform_skip             : 2;
+        /** QP delta within coding units.
+         *
+         * Allows setting cu_qp_delta_enabled_flag in the PPS.  When
+         * enabled, diff_cu_qp_delta_depth must also be set to a value
+         * allowed by the VAConfigAttribEncHEVCBlockSizes attribute.
+         */
+        uint32_t cu_qp_delta                : 2;
+        /** Weighted prediction.
+         *
+         * Allows setting weighted_pred_flag and weighted_bipred_flag in
+         * the PPS.  The pred_weight_table() data must be supplied with
+         * every slice header when weighted prediction is enabled.
+         */
+        uint32_t weighted_prediction        : 2;
+        /** Transform and quantisation bypass.
+         *
+         * Allows setting transquant_bypass_enabled_flag in the PPS.
+         */
+        uint32_t transquant_bypass          : 2;
+        /** Deblocking filter disable.
+         *
+         * Allows setting slice_deblocking_filter_disabled_flag.
+         */
+        uint32_t deblocking_filter_disable  : 2;
+        /* Reserved for future use. */
+        uint32_t reserved                   : 2;
+    } bits;
+    uint32_t value;
+} VAConfigAttribValEncHEVCFeatures;
+
+/** Attribute value for VAConfigAttribEncHEVCBlockSizes.
+ *
+ * This attribute describes the supported coding tree and transform block
+ * sizes of an HEVC/H.265 encoder configuration
+ */
+typedef union VAConfigAttribValEncHEVCBlockSizes {
+    struct {
+        /** Largest supported size of coding tree blocks.
+         *
+         * CtbLog2SizeY must not be larger than this.
+         */
+        uint32_t log2_max_coding_tree_block_size_minus3    : 2;
```

Comment:
ask a question: if some device support LTU = 64, but not support LTU=32. how to report this attributes?
how to distinguish it from : support LTU=64 , also support LTU=32. both ok.

Reply:
Only 64x64 luma CTBs supported:
`log2_max_coding_tree_block_size_minus3 = 3`
`log2_min_coding_tree_block_size_minus3 = 3`
Both 64x64 and 32x32 luma CTBs supported:
`log2_max_coding_tree_block_size_minus3 = 3`
`log2_min_coding_tree_block_size_minus3 = 2`